db/migrate/20230829201825_create_comments.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+# This migration creates the 'comments' table in the database.
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    # This block creates a new database table named 'comments'.
+    create_table :comments do |t|
+      # This line establishes a foreign key relationship.
+      # It associates the 'author_id' column with the 'users' table.
+      # The 'class_name' option specifies the related class, which is 'User'.
+      # 'foreign_key: true' adds a foreign key constraint.
+      t.belongs_to :author, class_name: 'User', foreign_key: true
+
+      # This line establishes a foreign key relationship.
+      # It associates the 'post_id' column with the 'posts' table.
+      # 'foreign_key: true' adds a foreign key constraint.
+      t.belongs_to :post, foreign_key: true
+
+      # This line defines the column for the content of the comment.
+      t.text :text
+
+      # This line defines the column for the creation timestamp of the comment.
+      t.datetime :created_at
+
+      # These lines add timestamps columns 'created_at' and 'updated_at'
+      # which automatically track when records are created and updated.
+      t.timestamps
+    end
+
+    # Add indexes to the foreign key columns for better performance
+    add_index :comments, :author_id
+    add_index :comments, :post_id
+  end
+end
+
+# Additional Migration for Creating Likes Table
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    # This block creates a new database table named 'likes'.
+    create_table :likes do |t|
+      # This line establishes a foreign key relationship.
+      # It associates the 'user_id' column with the 'users' table.
+      # The 'class_name' option specifies the related class, which is 'User'.
+      # 'foreign_key: true' adds a foreign key constraint.
+      t.belongs_to :user, class_name: 'User', foreign_key: true
```

Comment:
- [ Required ] I think it would be nice if the `user` id reference in the `likes` table is renamed to `author_id` the same way it is in the Entity relationship diagram. Currently, the column is named `user_id` which is not consistent with the project requirements. Please see the screenshot below and rectify this, remember to also update the code that adds an `index` to refer to `author_id` instead of `user_id`.

## Project screenshot
![Screenshot from 2023-08-31 22-04-22](https://github.com/kayjeee/Kayjeee_Postgresql_blogapp/assets/16392046/78608bb3-71b0-4356-af53-7e89dce4c766)